packages/orbit-components/src/TileGroup/TileGroup.stories.tsx

```diff
@@ -14,272 +15,265 @@ import TileGroup from ".";
 
 const getIcon = source => Icons[source];
 
-export default {
+type TileGroupPropsAndCustomArgs = React.ComponentProps<typeof TileGroup> &
+  React.ComponentProps<typeof Tile> & {
+    content: string;
+  };
+
+const meta: Meta<TileGroupPropsAndCustomArgs> = {
   title: "TileGroup",
+  component: TileGroup,
 };
 
-export const DefaultJustWrapper = ({ dataTest, as, content }) => {
-  return (
-    <TileGroup as={as} dataTest={dataTest}>
+export default meta;
+type Story = StoryObj<TileGroupPropsAndCustomArgs>;
+
+export const DefaultJustWrapper: Story = {
+  render: ({ content }) => (
+    <TileGroup>
       <Tile onClick={action("clicked")}>{content}</Tile>
```

Comment:
I would set `onClick={actions("clicked"))}` as a general `args` as it's used multiple times within the file.